src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1303,11 +1303,13 @@ public function getQueueableRelations()
         $relations = [];
 
         foreach ($this->getRelations() as $key => $relation) {
-            $relations[] = $key;
+            if (method_exists($this, $key)) {
+                $relations[] = $key;
+            }
 
             if ($relation instanceof QueueableCollection) {
-                foreach ($relation->getQueueableRelations() as $collectionKey => $collectionValue) {
-                    $relations[] = $key.'.'.$collectionKey;
+                foreach ($relation->getQueueableRelations() as $collectionValue) {
+                    $relations[] = $key.'.'.$collectionValue;
                 }
```

Comment:
Can you explain this change a bit more? Why did you make this change?

Reply:
As in Collection I am returning the result of `getQueueableRelations` instead of the previous `getRelations` it returns an integer-indexed array with the name of the relations from the first model in the collection., so I used the values instead of the keys.